packages/babel-cli/src/babel/util.js

```diff
@@ -2,7 +2,7 @@
 
 import readdirRecursive from "fs-readdir-recursive";
 import * as babel from "@babel/core";
-import includes from "lodash/includes";
+import includes from "lodash.includes";
```

Comment:
Can we replace this with Array.includes and eliminate the dep altogether?

https://github.com/jayaddison/babel/blob/dependencies/lodash-submodules/packages/babel-cli/src/babel/util.js#L50

Reply:
That seems reasonable - there are a few other places where `includes` is in use; are there any general rules for when it's safe to use the prototype function?  `babel-traverse` looks like it includes the `Array` type spec, if that's relevant:

https://github.com/jayaddison/babel/blob/dependencies/lodash-submodules/packages/babel-traverse/src/index.js#L93

Reply:
Should be safe to replace all the cases being used on an array, since node support for .includes is v6!